src/riak_pb_ts_codec.erl

```diff
@@ -131,6 +131,16 @@ decode_row([], Acc) ->
 decode_row([#tsrow{cells = Row} | T], Acc) ->
     decode_row(T, [decode_cells(Row, []) | Acc]).
 
+-spec decode_numeric(list()) -> float().
```

Comment:
Spec is wrong, we take a binary here.  
This pointed out an possible bug though. Here we expect binary back from the decode operation, [but when we encode](https://github.com/basho/riak_pb/pull/135/files#diff-5a85f13716f1297dfbd5fc46fd4c48ecR109), we give it a list.  What's it supposed to be?